packages/server/lambdas/alfresco/searchQueryBuilder/luceneQueryBuilder.ts

```diff
@@ -0,0 +1,78 @@
+import { IMimeSearchParameter, IModifiedSearchParameter, Paging, SearchParameter } from './types';
+
+const mimeTypesMapping = {
+  MSWORD: ['"application/msword"', '"application/vnd.openxmlformats-officedocument.wordprocessingml.document"'],
+  PDF: ['"application/pdf"'],
+};
+
+const DIVIDER = ':';
+const SEARCH_START = '+@cm\\' + DIVIDER;
+
+// TODO: from/to conversion to YYYY-MM-DD
```

Comment:
Maybe this TODO could get full use of library "date-fns" to format the date and couple with these 2 conditions:
- If given 1 year: then get first and last date of the year
- If given 2 years: then get first date of first year and last date of last year

Reply:
Just added date-fns, but your condition suggestions are something I hadn't thought of. Will look into those!